TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/console_link/models/container_utils.py

```diff
@@ -0,0 +1,25 @@
+import logging
+from pathlib import Path
+
+logger = logging.getLogger(__name__)
+VERSION_PREFIX = "Migration Assistant"
+
+
+def get_version_str() -> str:
```

Comment:
Can you also add a `/system/version` api that will return this value in the API?